src/main/java/org/bonitasoft/web/client/api/AuthenticationApi.java

```diff
@@ -38,45 +38,19 @@ public interface AuthenticationApi extends ApiClient.Api {
      * 
      * @param username the username (required)
      * @param password the password (required)
-     * @param tenant the user tenant (optional)
      * @param redirect \\\&quot;true\\\&quot; or \\\&quot;false\\\&quot;. \\\&quot;false\\\&quot; indicates that the service should not redirect to Bonita Portal
      *        (after a successful login) or to the login page (after a login failure). (optional, default to false)
      * @param redirectURL the URL of the page to be displayed after login (optional, default to )
+     * @return Response
      */
     @RequestLine("POST /loginservice")
     @Headers({
             "Content-Type: application/x-www-form-urlencoded",
             "Accept: application/json",
     })
-    void login(@Param("username") String username, @Param("password") String password, @Param("tenant") String tenant,
+    Response login(@Param("username") String username, @Param("password") String password,
```

Comment:
Why is the return type different?

Reply:
I order to be able to retrieve the response headers. It is a supported Feign feature.
This class is in the ignored by generation so that it is not overwriten.